umbraco-cloud/set-up/project-settings/umbraco-cicd/KnownLimitationsAndConsiderations.md

```diff
@@ -18,7 +20,22 @@
 
 * Given the necessity to avoid changes in other environments, the lack of strict coordination among multiple teams or individuals working on the same project elevates the risk of conflicts.
 
-As we continue to prototype and gather insights from beta users, there are some known limitations and considerations to be aware of:
+**Usage of Private NuGet Feeds**
+
+It is possible to reference private NuGet feeds [by using the Secrets Management feature](../../private-nuget-feed.md) on Umbraco Cloud. 
+
+If you plan on using this feature, the first time you are setting up the private feed configuration has to be done manually. The guide describes how to correctly set it up. The initial setup is not possible through the CI/CD pipeline.
+
+This is because Umbraco Cloud collects the private NuGet credentials from your leftmost environment. If your leftmost environment does not have the credentials, trying to push the NuGet.config changes through CI/CD will cause restore problems.
+
+You can also add the correct credentials that reference your Umbraco Cloud secrets by following the steps below:
+  1. Login to Kudu on your leftmost environment.
+  2. Navigate to your NuGet.config file via the console (the default location is the repository root).
+  3. Edit the NuGet.config file to include your private feed and credentials.
+  4. Save the file.
+  5. Commit the changes while in Kudu.
+
+After making sure that your NuGet.config has the credentials for your private feed, copy the NuGet.config file to your local project. You should be able to trigger the CI/CD flow successfully afterwards.
```

Comment:
at least in support, we never recommend out clients to make any changes in Kudu so they do not get used to making those changes there and then after a push from local or previous environment for the changes to be overwritten/removed as they were not committed. Wouldn´t it better in this case to use the same steps as https://docs.umbraco.com/umbraco-cloud/set-up/private-nuget-feed#step-4-add-private-myget-feed-on-umbraco-cloud to just make this change to the  NuGet.config file  locally and push it up to cloud environment? 

@jesp209i can I have your opinion on this as well?